### PR TITLE
docs(*) Fix some typos

### PR DESCRIPTION
**Pull request by pushkyn**
**NOTE**: Please read the CONTRIBUTING.md guidelines before submitting your patch,
and ensure you followed them all:

https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md

### Summary

Fix spelling

### Full changelog

* Edited docs to fix spelling

### Checklist:

- [x] [Commit message & atomicity](https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#commit-atomicity) checked
- [N/A] Documentation 
- [x] Spellchecked my updates
- [x] Ready to be merged

### CODE REVIEW

File: app/0.12.x/admin-api.md [hunk 1]
```diff
@@ -167,9 +167,9 @@ HTTP 200 OK
 
 ### Retrieve node status
 
-Retrieve usage information about a node, with some basic information about the connections being processed by the underlying nginx process, and the status of the database connection.
+Retrieve usage information about a node, with some basic information about the connections being processed by the underlying Nginx process, and the status of the database connection.
```
Comment (coopr) [inline app/0.12.x/admin-api.md]: We try to follow the advice from NGINX (the company) when spelling nginx (the product) - see https://forum.nginx.org/read.php?2,234083,234085#msg-234085

File: app/_hub/kong-inc/serverless-functions/index.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ different priority in the plugin chain.
     ...
     ```
 
-6. Test that our lua code will terminate the request when no header is passed:
```
Comment (coopr) [inline app/_hub/kong-inc/serverless-functions/index.md]: @hishamhm @thibaultcha do you prefer `lua` or `Lua`?

Comment (thibaultcha) [reply app/_hub/kong-inc/serverless-functions/index.md]: Please use `Lua`.

Comment (thibaultcha) [reply app/_hub/kong-inc/serverless-functions/index.md]: Also see the "What's in a name?" section at https://www.lua.org/about.html

File: app/0.12.x/admin-api.md (hunk 1 above)
Comment (pushkyn) [reply app/0.12.x/admin-api.md]: Got it, reverted changes

**Comment by coopr** [comment]
Good stuff, thanks very much @pushkyn !

**Pull request merged**